src/sentry/tasks/integrations.py

```diff
@@ -37,12 +37,13 @@ def post_comment(external_issue_id, data, **kwargs):
     )
     if installation.should_sync('comment'):
         installation.create_comment(
-            external_issue.key, data['text'])
+            external_issue.key, '%s wrote:\n\n%s' % (user.name, data['text']))
```

Comment:
If the goal is to have a block quote, each integration providing issues will likely have a different format. I know that GitHub uses Markdown while [Jira](https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=all) uses `bq. ` or `{quote}` blocks.

Reply:
oh sigh. Good catch @markstory . I wouldn't have thought to look.

Reply:
What if we move the message formatting logic into the `create_comment` function. That way each individual integration can be responsible for formatting it appropriately.

The task passes the raw components it knows the integration needs, but leaves the specifics of it to the integration itself. This invoking code becomes dumber (a good thing 😛).